common/src/main/java/net/william278/huskhomes/command/RtpCommand.java

```diff
@@ -140,6 +145,37 @@ private void executeRtp(@NotNull OnlineUser teleporter, @NotNull CommandUser exe
         // Generate a random position
         plugin.getLocales().getLocale("teleporting_random_generation")
                 .ifPresent(teleporter::sendMessage);
+
+        if (plugin.getSettings().getCrossServer().isEnabled()
+                && plugin.getSettings().getCrossServer().getBrokerType() == Broker.Type.REDIS) {
+            List<String> allowedServers = plugin.getSettings().getRtp().getAllowedServers();
+            String randomServer = allowedServers.get(random.nextInt(allowedServers.size()));
+            if (randomServer.equals(plugin.getServerName())) {
+                performLocalRTP(teleporter, executor, world, args);
+                return;
+            }
+            Message.builder()
+                    .type(Message.Type.REQUEST_RTP_LOCATION)
+                    .scope(Message.Scope.SERVER)
+                    .target(randomServer)
+                    .payload(Payload.withRTPRequest(Payload.RTPRequest.of(teleporter.getUsername(), world.getName())))
+                    .build().send(plugin.getMessenger(), teleporter);
+            return;
+        }
+
+        performLocalRTP(teleporter, executor, world, args);
+    }
+
+    /**
+     * Performs the RTP locally.
+     *
+     * @param teleporter person to teleport
+     * @param executor the person executing the teleport
+     * @param world the world to teleport to
+     * @param args rtp engine args
+     */
+    private void performLocalRTP(@NotNull OnlineUser teleporter, @NotNull CommandUser executor, @NotNull World world,
```

Comment:
For consistency, can we go with "Rtp" instead of "RTP"

Reply:
done